pkgs/development/libraries/gdcm/default.nix

```diff
@@ -60,11 +58,8 @@ stdenv.mkDerivation rec {
     zlib
   ] ++ lib.optionals enableVTK [
     vtk
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    ApplicationServices
-    Cocoa
-    libiconv
-  ] ++ lib.optionals enablePython [ swig python ];
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk ]
```

Comment:
(Unnecessary, as noted on the `vtk` PR.)

Reply:
Jep, removed it now as well.